.pre-commit-config.yaml

```diff
@@ -11,3 +11,4 @@ repos:
     -   id: fmt
     -   id: cargo-check
     -   id: clippy
+        args: ["--", "-D", "warnings", "-A", "clippy::result_large_err"]
```

Comment:
I _think_ there was a reason the `clippy` hook was split off from the others in the protocol repo but unfortunately I don't remember. This seems fine until we either find out the reason or don't.

Reply:
Clippy is definitely running right now, so will leave it as is until that reason inevitably pops up!